### PR TITLE
Create TextInputCompoent and add corresponidng color styles

### LINKED ISSUE #40: Text Input component
# REQUIREMENTS
We need a text input component in order to get user data in form and also to look for directions in main screen.
# DESIGN
image
image
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Must be like the provided design.
- [ ] Must have 4 states (disabled, default, active and error)

### PR DESCRIPTION
**Pull request by Ing-Heber**
# Description
I created a textInput component with four states (default, active, error and disabled) with corresponding styles in order to finish issue #40 . As you can see the component uses props to change the corresponding style 
 
 Examples:
I have added textInputType props to call the corresponding style as you can see in the Screenshot section if you write t
image you will see the search bar.

 # Testing
1. Download repo.
2. Go to project folder (tribomobile). 
3. Switch to my branch (feature/40-text-input-component).
4. Git pull and npm install if necessary.
5. You will se welcomeGuestScreen so you have to import my component image
6. Then Call the component and see the 5 types. 
  ## Screenshots 
image
image
image
image

### CODE REVIEW

**Review by alcamk — DISMISSED** [review]
Nice, hope this can work well on iOS

File: tribomobile/components/TextInputs.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      default:
+        styleProperties = [styles.container, styles.inputTextForm];
+        break;
+    }
+
+    return styleProperties;
+  };
+
+  return (
+    <>
```
Comment (OswaldoPineda) [inline tribomobile/components/TextInputs.js]: due you only have one component to return you don't need this tag

**Review by OswaldoPineda — DISMISSED** [review]

File: tribomobile/components/TextInputs.js (hunk 1 above)
Comment (LeyaimMagma) [reply tribomobile/components/TextInputs.js]: yep, you can return without using fragments <></>

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by JordiGN — DISMISSED** [review]
Col job men and the difference between the states is great

**Review by LeyaimMagma — DISMISSED** [review]

**Review by develiasdaniel — DISMISSED** [review]
Nice Design Heber!

**Review by tovl1 — DISMISSED** [review]
Good!

**Review by Ocosmee — APPROVED** [review]
Good job !

**Review by JordiGN — APPROVED** [review]
Nice design Heber, amazing job! Hope we can finally get it merged hahaha

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**